playground/streaming/benchmark.py

```diff
@@ -225,7 +225,7 @@
     args.agent_num_cycles = args.all_num_cycles
     args.synthesizer_num_cycles = args.all_num_cycles
 
-if args.create_graphs:
+if args.create_graphs or args.just_graphs:
```

Comment:
why the duplicated command line arguments?

Reply:
Create graphs will run all the benchmarking commands and then create graphs. Just graphs will skip all benchmarking and generate the graphs. It's useful if you want graphs but forgot to add the --create_graphs option when benchmarking.